aws-servicecatalog-serviceaction/src/main/java/software/amazon/servicecatalog/serviceaction/CreateHandler.java

```diff
@@ -13,34 +10,27 @@
 
 public class CreateHandler extends BaseHandler<CallbackContext> {
 
-    private static final String CREATE_ERROR_MSG = "Unable to create service action because: %s";
-
     @Override
     public ProgressEvent<ResourceModel, CallbackContext> handleRequest(
-        final AmazonWebServicesClientProxy proxy,
-        final ResourceHandlerRequest<ResourceModel> request,
-        final CallbackContext callbackContext,
-        final Logger logger){
+            final AmazonWebServicesClientProxy proxy,
+            final ResourceHandlerRequest<ResourceModel> request,
+            final CallbackContext callbackContext,
+            final Logger logger){
 
         ActionController actionController = ActionController
                 .builder()
                 .logger(logger)
                 .proxy(proxy)
                 .scClient(SCClientBuilder.getClient())
                 .build();
-        final ResourceModel desiredModel = request.getDesiredResourceState();
```

Comment:
If return is inside try catch there's no need to remove final.

Reply:
Updated code